getting_started.Rmd

```diff
@@ -132,14 +132,43 @@ When updates are made to files that aren't specific to the course content but in
     - [File for the OTTR_Quizzes updates](https://github.com/jhudsl/OTTR_Quizzes/blob/main/.github/sync.yml)
 
+#### Making edits to these files if you're not part of the jhudsl GitHub organization 
+
+If you're not part of the jhudsl GitHub organization, to edit the `sync.yml` file you'll need to
+  - fork the repo
+  - make the changes (described below) to `sync.yml`, committing them to a branch
+  - create a PR in the original OTTR repo using your branch
+  
+Detailed instructions for these steps are provided below:
+  - Click the pencil in the upper right for "Fork this repository and edit the file"
+  - Click the green button that says "Fork this repository"
+  - Make the edits to the file, adding your repository's name where it says `#NEW REPO HERE#`. Be careful to indent the same amount as the other repositories listed.
+![](https://raw.githubusercontent.com/jhudsl/OTTR_Template/main/resources/screenshots/edit-sync.yml.png)
+  - Click the green "Commit changes..." button. As the banner on the top of the page says, this will write your changes to a new branch in your fork so you can open a pull request.
+  - Add a commit message or extended description beyond what's there if you want to provide more info and click the green Propose Changes button.
+  - This will automatically open a new page comparing the changes from your branch to the original repo. Click the green "Create pull request" button to create a PR with your edits to the sync.yml file to the OTTR repo.
+  - Edit the title or provide an extended description beyond what's there if you want (if you already did that when you created your commit, GitHub will copy that over). Click the green "Create pull request" button. This will create a PR in the OTTR repo.
+  - To make sure that we see the PR, you'll want to add a comment with `@cansavvy` to alert us there's a PR that needs to be merged. Scroll down to "Add a comment", type the `@cansavvy`, and click the green "Comment" button.
+  
+Some alternative information on these steps are available [here from GitHub](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork) or from [James Priest's Gist on the Fork and PR Workflow](https://gist.github.com/james-priest/74188772ef2a6f8d7132d0b9dc065f9c) (note: you don't need to clone the forked repo locally, instead making edits online and committing to a new branch).
+
+If you'd rather not create a fork and open a PR, we can make edits to the `sync.yml` file for you if you open an issue in the relevant repo (links in table below) providing us the information about your repo (organization or username as well as the new repo name) or submit a response in our [OTTR Feedback Google Form](https://forms.gle/jGQnd5oemHWyuUq28) providing the information for your new repo in the "Do you have any overall suggestions for OTTR?" question.
```

Comment:
Can we move this earlier so people know they don't *have* to make a fork if they don't want to. I'd hate for them to follow all the steps and then get to this point and realize they didn't have to.